src/main/java/com/bootcamp/demo/restapi/ScooterController.java

```diff
@@ -0,0 +1,58 @@
+package com.bootcamp.demo.restapi;
+
+import com.bootcamp.demo.model.Scooter;
+import com.bootcamp.demo.service.ScooterService;
+import com.bootcamp.demo.service.exception.ServiceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
+
+@RestController
+@RequestMapping(path = "/api/scooters")
+public class ScooterController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ScooterController.class);
+    private final ScooterService scooterService;
+
+    public ScooterController(ScooterService scooterService) {
+        this.scooterService = scooterService;
+    }
+
+    @GetMapping
+    @ResponseBody
+    public ResponseEntity<Set<Scooter>> getScooters() {
+        Set<Scooter> scooters = null;
```

Comment:
No need to use an intermediate variable for this approach. You could simplify this to sth like:
```
try {
    return new ResponseEntity<>(scooterService.findAllScooters(), HttpStatus.OK);
} catch (ServiceException e) {
    return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
}
```
which is similar to what you have done in `getScooterById`, with the difference that the find call is inlined.

_Just as a note:_
Personally I find it easier to have only one return statement per method, but for that you would need to have sth like
```
Set<Scooter> scooters = null;
var status = HttpStatus.OK;
try {
    scooters = scooterService.findAllScooters();
} catch (ServiceException e) {
    status = HttpStatus.BAD_REQUEST;
}
return new ResponseEntity<>(scooters, status);
```
and in this case it is probably not very intuitive.